src/css/style.css

```diff
@@ -91,24 +91,71 @@ p {
 }
 
 .main-text {
-  border-radius: var(--default-radius);
-  border: solid var(--white) 1px;
-  padding: 30px 30px 20px 30px;
   text-align: left;
 }
 
 .main-text-title {
+  text-align: center;
   font-size: 1.375rem;
   padding: 0 0 20px 0;
 }
 
 .main-text-body {
-  background: url(/images/main-text-icons.svg) bottom right no-repeat;
-  background-size: 100px;
+  border-image: url(images/borders/border.svg) 10 10 10 10 stretch stretch;
+  border-style: solid;
+  border-width: 10px 10px 10px 10px;
+  padding: 30px 30px 20px 30px;
   font-size: 1.25rem;
   padding-bottom: 40px;
 }
 
+.hand-drawn-border {
```

Comment:
This seems simple and clear way to implement. Small part of me thinks to bring inline in the Elm since it's so "formulaic" but I think this is good and will be easy enough to maintain / tweak from here.